salt/utils/platform.py

```diff
@@ -177,3 +177,14 @@ def is_fedora():
         x.strip('"').strip("'") for x in linux_distribution()
     ]
     return osname == "Fedora"
+
+
+@real_memoize
+def is_photonos():
+    """
+    Simple function to return if host is Photon OS or not
+    """
+    (osname, osrelease, oscodename) = [
+        x.strip('"').strip("'") for x in linux_distribution()
+    ]
+    return osname == "VMware Photon OS"
```

Comment:
Based on the port I've been picking away at in #58856, it looks like this might be just `"VMware Photon"` pre-3.0, though 
I didn't go check that.  I don't know if the right answer is to check for that too, or just draw the support line at 3.0.

Reply:
Right now we're only testing on Photon OS 3, my opinion is that we should support that latest until we hear differently.